File: be_prepared/ruby/be_prepared.rb
def ask(prompt)
    print prompt
    gets
end

temp_f = ask("Enter Temperature in degrees F : ").to_f
temp_c = (temp_f - 32)*5/9
puts "Temperature Degrees C: #{temp_c}"
 
if temp_f > 100
    puts "Hydrate!!"
end

if temp_f < 32
    puts "Pack Long Underwear!!"
end

File: be_prepared/js/be_prepared.mjs
import readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Enter Temperature in degrees F : ", (answer) => {
    const tempF = parseFloat(answer);
    const tempC = (tempF - 32)*5/9;
    console.log("Temperature Degrees C: " + tempC);
    
    if(tempF > 100) {
        console.log("Hydrate!!");
    }

    if(tempF < 32) {
        console.log("Pack Long Underwear!!");
    }

    process.exit(0);
});

